build.gradle.kts

```diff
@@ -3,11 +3,14 @@
 import org.beryx.jlink.JlinkZipTask
 import org.gradle.jvm.tasks.Jar
 
+defaultTasks("licenseFormat")
+
 plugins {
     `java-library`
     application
     id("org.javamodularity.moduleplugin") version "1.8.12"
     id("org.openjfx.javafxplugin") version "0.1.0"
+    id("org.cadixdev.licenser") version "0.6.1"
```

Comment:
just a side note, if the plugin is applied in `allprojects` block, there's no need to apply it here, you could add infix `apply = false`

Reply:
![image](https://github.com/user-attachments/assets/a5b5f849-8e94-4abd-a0ae-27b981b4473c)
i dont think so

Reply:
That's just a pure gradle moment lol...

Reply:
Or webbrowser vscode moment